db/schema.rb

```diff
@@ -126,7 +126,9 @@
     t.string   "affiliation"
     t.integer  "team_captain_id"
     t.integer  "division_id"
+    t.integer  "users_count",     default: 0
```

Comment:
The migration for this field is missing which will cause problems in production.